pontoon/uxactionlog/views.py

```diff
@@ -0,0 +1,35 @@
+from django.contrib.auth.decorators import login_required
+from django.db import transaction
+from django.http import JsonResponse
+from django.views.decorators.http import require_POST
+
+from pontoon.base.utils import require_AJAX
+from pontoon.uxactionlog import forms, utils
+
+
+@login_required(redirect_field_name="", login_url="/403")
+@require_POST
+@require_AJAX
+@transaction.atomic
+def log_ux_action(request):
+    """Save a new UX action in the database."""
+    form = forms.UXActionLogForm(request.POST)
+
+    if not form.is_valid():
+        return JsonResponse(
+            {
+                "status": False,
```

Comment:
The `status` part of the response is a duplicate of the http status, I'd just choose one of the two.

Reply:
True... but we do this in pretty much every `JsonResponse` call, so I'd rather not break this "convention" partially.